pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[project]
+name = "sense_EEG"
+dynamic = ["version"]
+dependencies = []
+requires-python = ">=3.11"
```

Comment:
ROS2 humble in the docker image ships with `python 3.10`. Can we stick to that to avoid mixing python capabilities? We can then migrate to a newer version once the computer vision SDK uses it (soon!)